src/test/resources/fixtures/hmh/hmh-insighted/caliperGradeEventScore.described.json

```diff
@@ -8,15 +8,15 @@
       "type": "GradeEvent",
       "id": "urn:uuid:{uuid is auto-generated}",
       "actor": {
-        "id": "urn:uuid:{uuid of the student who completed the assignment}",
-        "type": "Person"
+        "id": "urn:uuid:{uuid of the teacher who graded the assignment} | {software application that auto-graded the assignment}",
```

Comment:
This is always the student id - as the grade event is for the student and not the teacher. As for the actor having this software application id - technically by caliper spec this is allowed but Ed Analytics implementation has chose for it to be of type Person only and for it to be student only.

Reply:
The above change came about on Padraig's request.
Do we need to get together and clarify this?

Reply:
From the Caliper spec:
For auto-graded scenarios the SoftwareApplication MUST be specified as the actor.
It isn't just allowed, it is required...

Reply:
The Grade Event is not 'for the student'.
The GradeEvent is recording an event which occurred in the system, and the actor who was involved in that event (the grading process) was either the assessment engine which graded the assignment, or the scorer (typically the teacher) who manually graded the assignment.
The assignment was assigned to a particular student (which is recorded by the assignee property of the Attempt).
For some other assessment events the actor is the student, but in that case also the Attempt would have an assignee property which was the student.

Reply:
By the way, there has been some discussion of adding a new event, possibly called MasteryEvent which would be student centric and would model it in terms of the student being the actor who masters a learning objective earning an attempt with a score (generated) and the scoredBy would be the scorer (human or SoftwareApplication).
But that is not going to be even thought about until after 1.2 is bedded down